api/versionpb/version.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package versionpb;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/descriptor.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+extend google.protobuf.MessageOptions {
+  optional string etcd_version_msg = 50000;
```

Comment:
Can we name the option in all the 3 extensions the same way
e.g. (`since_etcd_version=...`)
such that from user (dev) perspective it doesn't matter which of the 3 extensions is being used internally ?

Reply:
Based on my knowleage the answer is no. I will do some more research, but I don't think we can have same name for options of different types.